Packs/Active_Directory_Query/ReleaseNotes/1_1_13.md

```diff
@@ -0,0 +1,15 @@
+
+#### Integrations
+##### Active Directory Query v2
+- Added the option to get user by ***sAMAccountName*** in ***ad-get-user*** command.
+- Changed *CN* generation option, it will be generated by first and last name in ***iam-create-user*** command.
```

Comment:
```suggestion
- Updated the ***iam-create-user*** command to make sure the generated CN is unique.
```

Reply:
We checked before that it's a unique one, we changed the way it's mapped and I think we should clarify it, wdut?